japanese/lookup.py

```diff
@@ -102,7 +119,10 @@ def _field(name):
     try:
         return mw.reviewer.card.note()[name]
     except:
-        return
+        try:
+            return mw.reviewer.card.note().items()[name][1]
+        except:
+            return
```

Comment:
The new way to connect actions discards the default argument of onLookupExpression. I will fix this by connecting a lambda to the "...expression on alc" action instead (and roll back this change).